lib/stamp/emitters/am_pm.rb

```diff
@@ -1,22 +1,22 @@
 module Stamp
   module Emitters
     class AmPm
-      include Modifiable
-
-      AM = 'am'
-      PM = 'pm'
-
-      def initialize(&block)
-        @modifier = block
+      # @param [Hash] options
+      # @option options [Boolean] :upcase (default: false) uppercase meridian
```

Comment:
I have failed you. It's "meridiem", not "meridian". http://www.grammarphobia.com/blog/2014/03/ante-meridiem-vs-antemeridian.html

Maybe we should rename Emitters::AmPm emitter to Emitters::Meridiem?